sunpy/instr/rhessi.py

```diff
@@ -463,10 +463,8 @@ def backprojection(calibrated_event_list, pixel_size=(1., 1.) * u.arcsec,
 
     Examples
     --------
-    >>> import sunpy.data
     >>> import sunpy.data.sample
```

Comment:
I don't think you need to skip any of these anymore because it will already find the files locally and will not download them.